Source/WebCore/Modules/fetch/FetchBodyConsumer.cpp

```diff
@@ -126,6 +126,15 @@ static std::optional<MimeType> parseMIMEType(const String& contentType)
     return {{ WTFMove(type), WTFMove(subtype), parseParameters(StringView(input), semicolonIndex + 1) }};
 }
 
+FetchBodyConsumer::FetchBodyConsumer(Type type)
+    : m_type(type)
+{
+}
+
+FetchBodyConsumer::FetchBodyConsumer(FetchBodyConsumer&&) = default;
+FetchBodyConsumer::~FetchBodyConsumer() = default;
+FetchBodyConsumer& FetchBodyConsumer::operator=(FetchBodyConsumer&&) = default;
```

Comment:
Nit: I think some version of GTK will fail with default assignment operator, but I feel that the GTK port should add an #if/def for their old GCC version that lacks support (unlike their EWS bot, which *does* support it).

Reply:
Agreed!